yarpl/test/Observable_test.cpp

```diff
@@ -638,3 +638,15 @@ TEST(Observable, CancelSubscriptionChain) {
 
   LOG(INFO) << "cancelled after " << emitted << " items";
 }
+
+TEST(Observable, RangeIteratorObservable) {
+  auto range_observable = make_ref<RangeObservable<int>>(10, 15);
```

Comment:
what is the difference between this and 
Observables::range(...)?

Reply:
This is a replacement for the implementation of Range which relies on the currenet `Flowable::create`, which is the weird one that requires the generator lambda return a tuple of (are we done yet, how many elements were sent).